components/camera/server.go

```diff
@@ -116,7 +116,7 @@ func (s *serviceServer) GetImages(
 	if err != nil {
 		return nil, errors.Wrap(err, "camera server GetImages could not call Images on the camera")
 	}
-	imagesMessage := make([]*pb.Image, 0, len(imgs))
+	imagesMessage := []*pb.Image{}
```

Comment:
Can we revert this file? Don't want to change the camera server.

Reply:
Done.